resources/js/photoinit.js

```diff
@@ -178,12 +175,14 @@ function initPhotoSwipeFromDOM (gallerySelector) {
     $('.pswp__button--mail').on('click touchstart', function (e) {
         e.preventDefault();
         e.stopPropagation();
-
-        const img = gallery.currItem.src.split('/').pop();
-
         photoBooth.toggleMailDialog(img);
     });
 
+    $('.pswp__button--download').on('click touchstart', function (e) {
+        e.preventDefault();

```

Comment:
we might also need `e.stopPropagation();` , see https://github.com/andreknieriem/photobooth/issues/185

Reply:
Done.